.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
### [OPTIONAL]

-  This change should have been required in the first review. Since it wasn't am marking it as optional but it will be great if you could do it.

- [ ] It will be good to add this file to the `.gitignore`. This will ensure that your repository is clean and professional.